vxquery-cli/src/main/java/org/apache/vxquery/cli/VXQuery.java

```diff
@@ -423,29 +442,33 @@ private static void timingMessage(String message) {
         @Option(name = "-showrp", usage = "Show Runtime plan.")
         private boolean showRP;
 
-        @Option(name = "-compileonly", usage = "Compile the query and stop.")
-        private boolean compileOnly;
+        // Optional (Not supported by REST API) parameters. Only used for creating a local hyracks cluster
```

Comment:
@prestoncarman: What I meant by that comment was, if the user is giving **ip** and **port** for an already running REST server, we have no control over these parameters then (because the current REST api specs don't support those parameters). Otherwise, if we are creating a local hyracks cluster, we have control over them.